packages/workspace/src/generators/ci-workflow/__snapshots__/ci-workflow.spec.ts.snap

```diff
@@ -186,32 +93,30 @@ on:
       - main
   pull_request:
 
-# Needed for nx-set-shas within nx-cloud-main.yml, when run on the main branch
 permissions:
   actions: read
   contents: read
 
 jobs:
   main:
-    name: Nx Cloud - Main Job
-    uses: nrwl/ci/.github/workflows/nx-cloud-main.yml@v0.13.1
-    with:
-      main-branch-name: main
-      number-of-agents: 3
-      init-commands: |
-        pnpm exec nx-cloud start-ci-run --stop-agents-after="build" --agent-count=3
-      parallel-commands: |
-        pnpm exec nx-cloud record -- pnpm exec nx format:check
-      parallel-commands-on-agents: |
-        pnpm exec nx affected --target=lint --parallel=3
-        pnpm exec nx affected --target=test --parallel=3 --ci --code-coverage
-        pnpm exec nx affected --target=build --parallel=3
-
-  agents:
-    name: Nx Cloud - Agents
-    uses: nrwl/ci/.github/workflows/nx-cloud-agents.yml@v0.13.1
-    with:
-      number-of-agents: 3
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      # Cache node_modules
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: 'npm'
+      - run: npm ci
+      - uses: nrwl/nx-set-shas@v4
+
+      # uncomment to enable task distribution
```

Comment:
Should we also advise people to remove the parallel 3 when they do this to avoid unnecessary limitations to throughput?

Reply:
(Applies in multiple locations)

Reply:
This strongly depends on the type of machine used, the complexity of the tasks, etc. Most of our customers still use `parallel=3`. This is something that should be documented in our recipes but not necessarily in the generated workflows.